src/main/java/org/reactome/addlinks/fileprocessors/HPAFileProcessor.java

```diff
@@ -0,0 +1,53 @@
+package org.reactome.addlinks.fileprocessors;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class HPAFileProcessor extends FileProcessor<List<String>>
+{
+
+    public HPAFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public HPAFileProcessor()
+    {
+        super(null);
+    }
+
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        try {
+            String dirToHPAFiles = this.unzipFile(this.pathToFile);
+            String inputFilepath = dirToHPAFiles + "/" + this.pathToFile.getFileName().toString().replace(".zip", "");
+
+            for (String line : Files.readAllLines(Paths.get(inputFilepath)).stream().skip(1).collect((Collectors.toList()))) {
```

Comment:
This is a bit long on an expression to have directly in the for loop.  It might be better to write:

```
for (String line : getLinesAfterHeader(inputFilepath)) {
...
}
```
and have the helper method return the lines you want using this logic.  It also makes it explicit why you're skipping the first line.